sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -0,0 +1,1299 @@
+## API Report File for "@azure/ai-form-recognizer"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+/// <reference types="node" />
+
+import { AzureKeyCredential } from '@azure/core-auth';
+import { KeyCredential } from '@azure/core-auth';
+import { OperationOptions } from '@azure/core-client';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PipelineOptions } from '@azure/core-rest-pipeline';
+import { PollerLike } from '@azure/core-lro';
+import { PollOperationState } from '@azure/core-lro';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public
+export type AnalysisPoller<Result = AnalyzeResult<AnalyzedDocument>> = PollerLike<DocumentAnalysisPollOperationState<Result>, Result>;
+
+// @public
+export interface AnalyzedDocument {
+    boundingRegions?: BoundingRegion[];
+    confidence: number;
+    docType: string;
+    fields: {
+        [field: string]: DocumentField;
+    };
+    spans: DocumentSpan[];
+}
+
+// @public
+export interface AnalyzeDocumentsOptions<Result = AnalyzeResult<AnalyzedDocument>> extends OperationOptions, PollerOptions<DocumentAnalysisPollOperationState<Result>> {
+    locale?: string;
+    pages?: string[];
+}
+
+// @public
+export interface AnalyzeResult<Document = AnalyzedDocument> {
+    apiVersion: FormRecognizerApiVersion;
+    content: string;
+    documents: Document[];
```

Comment:
`documents` here hold the results not the actual documents right? should another name be considered here that emphasizes this?

Reply:
Same comment applies for the type variable name as well.